controller/error_pages.rst

```diff
@@ -303,7 +303,12 @@ In that case, you might want to override one or both of the ``showAction()`` and
 
     The :ref:`error page preview <testing-error-pages>` also works for
     your own controllers set up this way.
+    
+.. note::
 
+    The above only applies to websites created using symfony/website-skeleton and not symfony/skeleton. See
```

Comment:
That's not accurate: the above applies to apps that have TwigBundle enabled!
that's perfectly doable after starting with a skeleton